pkg/executables/docker.go

```diff
@@ -70,3 +73,28 @@ func (d *Docker) AllocatedMemory(ctx context.Context) (uint64, error) {
 	totalMemory = totalMemory[1 : len(totalMemory)-2]
 	return strconv.ParseUint(totalMemory, 10, 64)
 }
+
+func (d *Docker) Pull(ctx context.Context, image string) error {
```

Comment:
I think we already have the method `DockerPullImage` for this.

Reply:
We can remove this and rename the already existing one to `PullImage` to match the `PushImage` naming below.

Reply:
done